src/main/scala/com/lohika/vteraz/data/CsvReaderImpl.scala

```diff
@@ -0,0 +1,18 @@
+package com.lohika.vteraz.data
+
+import scala.io.Source
+
+
+class CsvReaderImpl(filePath: String) extends CsvReader {
+
+  override def readFile(): List[String] = {
+    try {
+      Source.fromFile(filePath).getLines().toList
+    } catch {
+      case e: Exception => {
+        Console.err.println("Failed to read file: " + e.getMessage)
+        sys.exit(1)
```

Comment:
how you would unit test readFile function having sys.exit. I would catch exceptions in main method and there put sys.exit. sys.exit should not be spread over your code.